### DISCUSSION TITLE
Elyra survey questions (WIP)

### DISCUSSION THREAD

**Discussion by ptitzler** [elyra-ai/elyra#2239] [Internal]
Using this thread to collect input on what questions we want to ask our users in our first survey. (Please do not answer the questions; this is not the survey!) 
- One question per comment please to allow for easy replies to each question (e.g. if changes are required).
- Upvote a question if you think it is something we should ask. Since we probably want to limit our survey we might not get to ask every question listed below.

**Comment by ptitzler** [comment]
**Q: Which runtime environment are you using to run pipelines? (Check all that apply)**
- Local/JupyterLab
- Kubeflow Pipelines
- Apache Airflow

**Comment by ptitzler** [comment]
**Q: Which Elyra features are you using and how frequently are you using them? Range: 0 (not using) 10 (daily)** 

- Visual pipeline editor [0 ... 10]
- Python script editor [0 ... 10]
- R script editor [0 ... 10]
- Code snippets [0 ... 10]
- Run notebook or Python/R script as batch? [0 ... 10]
- ? [0 ... 10]

**Comment by ptitzler** [comment]
**Q: If you are running (or planning to run) Elyra pipelines on Kubeflow Pipelines or Apache Airflow ... (check all that apply)**
- I am using pipelines to run Jupyter notebooks or Python/R scripts ("generic components").
- I am using pipelines to run components/operators that I/my team/my company has created ("custom components").
- I am using pipelines to run components/operators that a third party has created ("custom components").

**Comment by ptitzler** [comment]
**Q: If you are running (or planning to run) Elyra pipelines on Kubeflow Pipelines or Apache Airflow  and use custom components; where are you typically storing the component definitions (YAML or Python files) ... (check all that apply or specify if not listed)**
- Local filesystem
- Source control system - GitHub
- Source control system - GitLab
- Source control system - other (please specify)
- Object storage
- URL addressable location
- Other (please specify)

**Comment by ptitzler** [comment]
**Q: How frequently do you use  ...? Range: 0 (don't use) 10 (frequently)**
- The [documentation](https://elyra.readthedocs.io/en/stable/) [0 ... 10]
- The [tutorials/examples](https://github.com/elyra-ai/examples) [0 ... 10]
- The [discussion forum on gitHub](https://github.com/elyra-ai/elyra/discussions) [0 ... 10]
- The [gitter community](https://gitter.im/elyra-ai/community) [0 ... 10]
- ... [0 ... 10]
-

**Comment by ptitzler** [comment]
**Q: How do you install/deploy Elyra ... (check all that apply)**
- [Install from PyPI]()
- [Install from conda-forge](https://elyra.readthedocs.io/en/stable/getting_started/installation.html#conda)
- [Install from source](https://elyra.readthedocs.io/en/stable/getting_started/installation.html#build-from-source)
- [Elyra container image](https://elyra.readthedocs.io/en/stable/getting_started/installation.html#docker)
- [Kubeflow Notebook server container image](https://elyra.readthedocs.io/en/stable/recipes/using-elyra-with-kubeflow-notebook-server.html)
- [Open Data Hub](https://elyra.readthedocs.io/en/stable/recipes/deploying-elyra-with-opendatahub.html)
- Other (please specify)

**Comment by kevin-bates** [comment]
**Q: Which pipeline platform(s) do you use?**

- Kubeflow Pipelines
- Apache Airflow
- Others?

**Q: If currently not supported, are you interested in integrating your own pipeline processor for that platform?**
